modules/apim_gateway/templates/carbon-home/bin/wso2server.sh.erb

```diff
@@ -33,7 +33,7 @@
 # OS specific support.  $var _must_ be set to either true or false.
 #ulimit -n 100000
 
-export JAVA_HOME="<%= @java_home %>"
+export JAVA_HOME="<%= @java_symlink %>"
```

Comment:
Any special reason to name it as "java-symlink" ?

Reply:
The parameter points to the path of a symlink pointing to JAVA_HOME